docs/self-managed/identity/orchestration-identity/configuration.md

```diff
@@ -0,0 +1,49 @@
+---
+id: configuration
+title: "Configuration"
+description: "Learn about the Identity configuration options available in your Orchestration cluster."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::note
+As a Spring Boot application, Identity supports any standard
+[Spring configuration](https://docs.spring.io/spring-boot/reference/features/external-config.html) method.
+:::
+
+The following variables apply globally to all components within the Camunda Orchestration core: Zeebe, Operate, and Tasklist.
+
+<!-- updates must be made to BOTH tables -->
+<Tabs>
+  <TabItem value="env" label="Environment variables" default>
+
+| Environment variable                                  | Description                                                                                       | Default value       |
+| ----------------------------------------------------- | ------------------------------------------------------------------------------------------------- | ------------------- |
+| `SPRING_PROFILES_ACTIVE`                              | **Note:** This property will be deprecated as additional authentication methods become available. | `consolidated-auth` |
+| `CAMUNDA_SECURITY_AUTHENTICATION_METHOD`              | The authentication method to use.                                                                 | `basic`             |
+| `CAMUNDA_SECURITY_AUTHENTICATION_UNAUTHENTICATED-API` | If the API is enabled without authentication.                                                     | `true`              |
+| `CAMUNDA_PERSISTENT_SESSIONS_ENABLED`                 | Enables shared authentication between the Orchestration web applications (Operate and Tasklist).  | `true`              |
+| `CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED`             | If authorizations are enabled.                                                                    | `true`              |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME`   | The username of the first user.                                                                   | `demo`              |
```

Comment:
Is this "first user" persistent after the initial configuration? Should the example really be "admin" or something that suggests elevated privileges? Or is this just a case where we need any kind of user to populate this table and get things moving?

Reply:
Good points @akeller - the is persisted in the deployment the choice of `demo` was to match the C7 behaviour IIRC where you have a initial user that can do anything without needing to perform additional setup